test_scripts/Defects/5_0/2385_Conditions_for_SDL_to_respond_SubscribeButtion_UnsubscribeButton_SUCCESS_to_mobile_app.lua

```diff
@@ -0,0 +1,74 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2385
+--
+-- Description:
+-- Conditions for SDL to respond SubscribeButtion/UnsubscribeButton (SUCCESS) to mobile app
+-- Precondition:
+--
+-- In case:
+-- 1) SDL successfully sends OnButtonSubscription to mqueue
```

Comment:
Please update description without mqueue, because openSDL does not send messages through this transport.